Framework/PythonInterface/plugins/algorithms/LoadCIF.py

```diff
@@ -76,8 +76,11 @@ def _getSpaceGroupFromString(self, cifData):
 
     def _getCleanSpaceGroupSymbol(self, rawSpaceGroupSymbol):
         # Remove :1 and :H from the symbol. Those are not required at the moment because they are the default.
-        removalRe = re.compile(':[1H]', re.IGNORECASE)
-        return re.sub(removalRe, '', rawSpaceGroupSymbol).strip()
+        # Also substitute 'R' and 'Z' endings used by ICSD to indicate alternative origin choice or settings
+        mappings = {':[1H]':'', ':[1h]':'', ' S$':'', ' H$':'', ' Z$':' :2', ' R$':' :r'}
```

Comment:
I think the first two could be merged into `':[1Hh]': ''`, but this is nothing major...just one regex less.